From dba5651181cc2630b0fa32d60d6195936f0c6bb0 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Wed, 5 Sep 2018 09:02:35 +0200
Subject: [PATCH 4/4] Merge tag 'android-9.0.0_r6' into
 lineage-16.0-android-9.0.0_r6

Android 9.0.0 Release 6 (PPR2.180905.005)

* tag 'android-9.0.0_r6':
  Version bump to PPR2.180905.005
  Version bump to PPR2.180905.004
  Version bump to PPR2.180905.003
  Version bump to PPR2.180905.002
  Updating Platform Security String to 2018-09-05 Bug: 111501777 (cherry picked from commit 6bc223c9af044ad06e2f1abc0c4570a7371f9a3a)

Change-Id: Ied7727225d7026f19ae8a63d85cc6885e35a2462
---
 core/build_id.mk         | 2 +-
 core/version_defaults.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index faabf57dc..59a9033a1 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=PPR1.180610.009
+export BUILD_ID=PPR2.180905.005
diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index af45d4499..3c1c3c913 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -231,7 +231,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2018-08-05
+      PLATFORM_SECURITY_PATCH := 2018-09-05
 endif
 
 ifndef PLATFORM_SECURITY_PATCH_TIMESTAMP
-- 
2.17.1

